cli/script/command-parser.ts

```diff
@@ -256,6 +256,16 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
 
         addCommonConfiguration(yargs);
     })
+    .command("link", "Link a new authentication provider to an account with the CodePush server", (yargs: yargs.Argv) => {
```

Comment:
"Link an additional authentication provider (e.g. GitHub) to an existing CodePush account"?